go/cmd/dolt/commands/dump_docs.go

```diff
@@ -34,6 +34,33 @@ const (
 		"title: CLI\n" +
 		"---\n\n" +
 		"# CLI\n\n"
+
+	globalArgumentsSection = `
+## Global Arguments
+
+Dolt subcommands are in transition to using the flags listed below as global flags.
+The sql subcommand is currently the only command that uses these flags. All other commands will ignore them.
```

Comment:
This is out of date, yeah?

Reply:
Can we define a var for where this is used in the main command and reference that here?